src/worldfactory.cpp

```diff
@@ -602,156 +598,20 @@ std::string worldfactory::pick_random_name()
     return get_next_valid_worldname();
 }
 
-int worldfactory::show_worldgen_tab_options(WINDOW *win, WORLDPTR world)
+int worldfactory::show_worldgen_tab_options(WINDOW*, WORLDPTR world)
```

Comment:
If you don't need that WINDOW argument, get rid of it.

Reply:
Can't that's what it expects:

```
typedef std::function<int( WINDOW *, WORLDPTR )> worldgen_display;
std::vector<worldgen_display> tabs;
```